.k8s.mk

```diff
@@ -26,7 +26,7 @@ DEPLOY_ICMP = $(DEPLOY_ICMP_KERNEL) $(DEPLOY_ICMP_VPP)
 DEPLOY_VPN = secure-intranet-connectivity vppagent-firewall-nse vpn-gateway-nse vpn-gateway-nsc
 DEPLOYS = $(DEPLOY_INFRA) $(DEPLOY_ICMP) $(DEPLOY_VPN)
 
-CLUSTER_CONFIGS = cluster-role-admin cluster-role-binding cluster-role-view
+CLUSTER_CONFIGS = cluster-role-admin cluster-role-binding cluster-role-view crd-networkservices crd-networkserviceendpoints crd-networkservicemanagers
```

Comment:
I'd split these into `CLUSTER_CONFIG_ROLE` an `CLUSTER_CONFIG_CRD` and then `CLUSTER_CONFIG = $(CLUSTER_CONFIG_ROLE) $(CLUSTER_CONFIG_CRD)` Makes thisngs more manageable.